src/attributes/codegen.md

```diff
@@ -49,10 +49,48 @@ r[attributes.codegen.cold]
 The *`cold` [attribute]* suggests that the attributed function is unlikely to
 be called.
 
+r[attributes.codegen.naked]
+## The `naked` attribute
+
+r[attributes.codegen.naked.intro]
+The *`naked` [attribute]* prevents the compiler from emitting a function prologue and
+epilogue for the attributed function.
```

Comment:
We're unwrapping lines on new things added, so go ahead and unwrap these all if you would.